packages/nextjs/components/Footer.tsx

```diff
@@ -3,7 +3,8 @@ import Link from "next/link";
 import { hardhat } from "viem/chains";
 import { MagnifyingGlassIcon } from "@heroicons/react/24/outline";
 import { HeartIcon } from "@heroicons/react/24/outline";
-import { SwitchTheme } from "~~/components/SwitchTheme";
+// SwitchTheme is commented out until it is fully implemented
```

Comment:
Let's remove the commented code. We have git if we want to rollback :D

Reply:
Deleted! I need to defeat my diogenes syndrome in those spots :)